public/docs/ts/latest/guide/structural-directives.jade

```diff
@@ -436,49 +456,50 @@ a#template
   It is never displayed directly. 
   In fact, before rendering the view, Angular _replaces_ the `<template>` and its contents with a comment.
   
-  If there is no structural directive, if you merely wrap some elements in a `<template>` and do nothing with it, 
+  If there is no structural directive, and you merely wrap some elements in a `<template>` and do nothing with it, 
```

Comment:
Ugh. I think the following is better ... we don't need to say "and do nothing with it"
>If there is no structural directive and you merely wrap some elements in a `<template>`, those elements disappear.